docs/writing-a-locustfile.rst

```diff
@@ -142,11 +142,9 @@ The easiest way to add a task for a User is by using the :py:meth:`task <locust.
 
 .. code-block:: python
 
-    from locust import User, task, constant
+    from locust import User, task
 
     class MyUser(User):
-        wait_time = constant(1)
```

Comment:
Again, using zero wait time in examples - especially if they're printing stuff - gives a horrible user experience for new users who copy & paste examples while familiarizing with Locust.

Reply:
Agreed, I tried to leave it in on those examples but missed this one. Fixed now in 81d97072413eccbe4e88c17f219c35dbe20e4bab